crates/pop-cli/src/commands/build/spec.rs

```diff
@@ -40,20 +38,20 @@ pub(crate) enum ChainType {
 	// A development chain that runs mainly on one node.
 	#[default]
 	#[strum(
-		serialize = "Development",
+		serialize = "development",
```

Comment:
Needs the capitalization. Otherwise the command seems to complete successfully.

Reply:
I don't understand. I changed this so that the user doesn't have to write "Development" but "development"

Reply:
Yeah, so I'm not really familiar with the details of strum here. 
But the reason of the failure is that the the node or spec builder requires this to be "Development". So the pain point is when this variant is serialized into the spec.

Although, even if we have `serialize = "Development"` users will be able to use `--type development` just fine. It's a bit confusing.

Reply:
Isnt there an ignore case flag on strum? https://docs.rs/strum/latest/strum/additional_attributes/index.html

Reply:
This seems resolved now. Is there anything else you want to do here, @Daanvdplas ?